horace_core/utilities/split_data_blocks.m

```diff
@@ -1,16 +1,18 @@
-function [chunks, cumulative_sum] = split_data_blocks(start_val,block_sizes, max_chunk_sum)
+function [chunks, cumulative_sum] = split_data_blocks(start_pos,block_sizes, buf_size)
 % Split the array of the data blocks into the blocks, which have equal or
 % almost equal size
 %
 % Input:
 % ------
-% start_val          A vector of numeric, non-negative, values describing a
-%                    position of a block on some media
+% start_pos          A vector of numeric, non-negative, values describing a
+%                    positions of blocks on some media
 % block_sizes        A vector of numeric, non-negative, values describing
 %                    block sizes, to process. Its assumed that the block is
 %                    continuously located on the media.
-% max_chunk_sum      A positive value specifying the maximum sum for each
+% buf_size           A positive value specifying the limiting size for each
```

Comment:
I would say the original is a better descriptor unless functionality has changed

Reply:
functionality has changed. Splitting into partial blocks appears to be noticeably more expensive than splitting at the edges of blocks (unless, of course size is so big that is not fitting the double buffer and needs to be split anyway)